### PR TITLE
Restructure Awesome books app code

### PR DESCRIPTION
**Pull request by wandergithub**
Restructured the preview Awesome books app code into modules:
1. Initialized directory and files.
2. Set linters.
3. Divided the all the code into different modules.
4. Maintained functionalities using arrow functions
5. Finally, added the luxon library, added script, imported and set the date/time on the page.

### CODE REVIEW

File: modules/removeBook.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+export default function removeBook(element) {
+  if (element.target.classList.contains('btn2')) {
+    const PARENT = element.target.parentNode;
+    const TITLE = PARENT.children[0].children[0].textContent;
+    const AUTHOR = PARENT.children[0].children[1].textContent;
+    PARENT.remove();
+    Storage.removeBook(TITLE, AUTHOR);
+  }
+}
```
Comment (EricMbouwe) [inline modules/removeBook.js]: - [x] Let's use `arrow` syntax for functions as this is a project requirement.

File: modules/createBooklist.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import BookList, { givenAuthor, givenName } from './BookList.js';
+
+export default function createBooklist() {
+  const booklist = new BookList(givenName.value, givenAuthor.value);
+  givenName.value = '';
+  givenAuthor.value = '';
+  BookList.addBook(booklist);
+}
```
Comment (EricMbouwe) [inline modules/createBooklist.js]: - [x] Let's use `arrow` syntax for functions as this is a project requirement.

File: modules/displayDate.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { DateTime } from './luxon.js';
+
+const now = DateTime.now();
+export default function displayDate() {
+  const container = document.querySelector('.date');
+  container.textContent = now.toLocaleString(DateTime.DATETIME_MED);
+}
```
Comment (EricMbouwe) [inline modules/displayDate.js]: - [x] Let's use `arrow` syntax for functions as this is a project requirement.

File: modules/sectionsHandler.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    list.classList.add('active');
+  } else if (link.classList.contains('add')) {
+    list.classList.remove('active');
+    contact.classList.remove('active');
+    add.classList.add('active');
+  } else if (link.classList.contains('contact')) {
+    list.classList.remove('active');
+    add.classList.remove('active');
+    contact.classList.add('active');
+  }
```
Comment (EricMbouwe) [inline modules/sectionsHandler.js]: - [x] Let's use `arrow` syntax for functions as this is a project requirement.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
### Hi @wandergithub,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @wandergithub 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [x] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**